index.js

```diff
@@ -63,6 +73,14 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+let weather = 'Rainy';
+let message;
+if (weather = 'Sunny') {
+    message = 'Yay! It\'s sunny outside'
+} else {
+    message = "Aww! It's not sunny outside"
+}
+console.log(message);
```

Comment:
you should be able to check the output of this file by running in your terminal on your computer
from inside the directory/folder for this repository `node index.js`

Also if it's easier to read the consoles/see what line you are logging, you can always provide additional arguments to the console.log. I usually do something like this..
```
console.log ('message output: ', message);